creepy/utils/tests/test_processify.py

```diff
@@ -0,0 +1,47 @@
+import os
+import time
+import signal
+import multiprocessing
+from threading import Thread
+
+import pytest
+
+from ..processify import processify
+
+
+def test_processify_on_identity_function():
+    for i in range(100):
+        assert i == processify(lambda: i)()
+
+
+def test_processify_child_crash():
+    for i in range(2):
+        with pytest.raises(RuntimeError, match=f'exited with code {i}'):
+            processify(quit)(i)
+
+
+def test_processify_parent_crash():
+    connection = multiprocessing.Queue()
+
+    @processify
+    def child():
+        connection.put(os.getpid())
+        time.sleep(100500)
+
+    @processify
+    def parent():
+        Thread(target=child, daemon=True).start()
+        time.sleep(0.1)  # wait a bit so child can enqueue its `pid`
+        os.kill(os.getpid(), signal.SIGKILL)  # suicide
+
+    with pytest.raises(RuntimeError, match=f'exited with code -{signal.SIGKILL}'):
+        parent()
+    child_pid = connection.get()
+    old_handler = signal.signal(signal.SIGALRM, lambda: 0 / 0)
```

Comment:
`0 / 0` could not throw an exception in some languages(at least in C++), so it's better to have `1 / 0` which is definitely exception.

Reply:
Ok. But in C++ there's no such exception even for 1 / 0. Python [raises it](https://docs.python.org/3/library/exceptions.html#ZeroDivisionError) if the second argument is 0.